Automattic/jetpack
# Jetpack Setup Wizard: Add recommended features calculation

#### Changes proposed in this Pull Request:

This PR adds the functionality that calculates the recommended features based on the user's answers in the setup wizard.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
Project: p1HpG7-8BF-p2
Design: pbtFPC-ol-p2

#### Testing instructions:






1. Add the following to the end of wp-config.php: add_filter( 'jetpack_show_setup_wizard', '__return_true' );
2. Visit `/wp-admin/admin.php?page=jetpack#/setup` and go through the flow.
3. Verify that when you get to `/wp-admin/admin.php?page=jetpack#/setup/features` the list of features displayed are selected according to your choices in the questionnaire. You can use the back button to easily re-test different combinations.

#### Proposed changelog entry for your changes:

* None, there should be one entry for the whole Setup Wizard
